### ISSUE TITLE
failed to cherry pick PR 4062 from cyclinder, to branch release-v0.8

### ISSUE THREAD

**Issue by weizhoublue** [spidernet-io/spiderpool#4077]
commits 603c57e04da85655a6f7672612f2463fad5cffda of cyclinder conflict when merging to branch release-v0.8, please manually cherry pick it by yourself. 
 PR <https://github.com/spidernet-io/spiderpool/pull/4062> ,  action <https://github.com/spidernet-io/spiderpool/actions/runs/10933291536>  

  ```
Auto-merging pkg/coordinatormanager/coordinator_validate.go
Auto-merging pkg/multuscniconfig/multusconfig_mutate.go
Auto-merging test/doc/spidermultus.md
CONFLICT (content): Merge conflict in test/doc/spidermultus.md
Auto-merging test/e2e/spidermultus/spidermultus_test.go
CONFLICT (content): Merge conflict in test/e2e/spidermultus/spidermultus_test.go
error: could not apply 603c57e0... Merge pull request #4062 from cyclinder/spidermultusconfig/fix_panic
hint: After resolving the conflicts, mark them with
hint: "git add/rm <pathspec>", then run
hint: "git cherry-pick --continue".
hint: You can instead skip this commit with "git cherry-pick --skip".
hint: To abort and get back to the state before "git cherry-pick",
hint: run "git cherry-pick --abort".
hint: Disable this message with "git config advice.mergeConflict false"
```

**Comment by weizhoublue** [comment]
@cyclinder

**Comment by ty-dc** [comment]
fixed #4099
